.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: minimal
+
+services:
+  - docker
+
+env:
+  - HAZELCAST_IMAGE=hazelcast-enterprise
+  - HAZELCAST_IMAGE=hazelcast-oss
+
+install:
+  - docker build -t $HAZELCAST_IMAGE $HAZELCAST_IMAGE
+
+script:
+  - docker images
+  - |
+    if [ "$HAZELCAST_IMAGE" = "hazelcast-oss" -o -n "$HZ_LICENSE_KEY" ]; then
+      docker run --name ${HAZELCAST_IMAGE}-container -e HZ_LICENSE_KEY=$HZ_LICENSE_KEY -d --rm $HAZELCAST_IMAGE
+      sleep 10
+      HZ_IP=$(docker inspect --format '{{ .NetworkSettings.IPAddress }}' ${HAZELCAST_IMAGE}-container)
```

Comment:
Would be better to expose the port instead of digging for the container ip

Reply:
Unfortunately not in combination with the subsequent simple NetCat check. The exposed port will be open even if there is nothing listening on the target port in container.